mixer/test/client/istio_authn_origin_jwt_bound_peer/authn_report_test.go

```diff
@@ -35,24 +49,29 @@ const authnConfig = `
           }
         }
       ]
-    },
-    "jwt_output_payload_locations": {
-      "issuer@foo.com": "sec-istio-auth-jwt-output"
     }
   }
 `
 
 const secIstioAuthUserInfoHeaderKey = "sec-istio-auth-jwt-output"
 
-const secIstioAuthUserinfoHeaderValue = `
-{
-  "iss": "issuer@foo.com",
-  "sub": "sub@foo.com",
-  "aud": "aud1",
-  "non-string-will-be-ignored": 1512754205,
-  "some-other-string-claims": "some-claims-kept"
-}
-`
+const secIstioAuthUserinfoHeaderValue = `{"aud":"aud1","exp":20000000000,` +
+	`"iat":1500000000,"iss":"issuer@foo.com","some-other-string-claims":"some-claims-kept",` +
+	`"sub":"sub@foo.com"}`
+
+// jwt is formed from the value in secIstioAuthUserinfoHeaderValue
+const jwt = "eyJhbGciOiJSUzI1NiIsImtpZCI6IkRIRmJwb0lVcXJZOHQyenBBMnFYZ" +
```

Comment:
can we put these shared strings into a file to be shared?

Reply:
Done.